sdk/schema/fieldname/names.go

```diff
@@ -7,6 +7,7 @@ const (
 	APIHost         = sdk.FieldName("API Host")
 	APIKey          = sdk.FieldName("API Key")
 	APIKeyID        = sdk.FieldName("API Key ID")
+	APIKeySecret    = sdk.FieldName("API Key Secret")
```

Comment:
Could we reuse `API Secret` as the fieldname because that seems to be [referenced in the config file generated as well](https://docs.spacelift.io/integrations/api#authenticating-with-the-api)?

<img width="734" alt="image" src="https://github.com/1Password/shell-plugins/assets/18581859/49a94db5-553c-45ae-9c03-ebd5e88a0b3d">

Reply:
I went with `API Key Secret` as it's the full internal name for it, but I see that from user's point it may be not clear that those are the same. I'll change it.